target/linux/mediatek/base-files/lib/preinit/05_set_preinit_iface

```diff
@@ -13,6 +13,12 @@ set_preinit_iface() {
 		ip link set eth0 up
 		ifname=eth0
 		;;
+	nradio,c8-668gl)
+		/* eth1 is disabled to ensure the RM520N-GL 5G module registers as eth1.*/
+		ip link set eth0 up
+		ip link set eth1 down
+		ifname=lan1
```

Comment:
why do this?
eth1 should up and nothing related to RM520N-GL
RM520N-GL never register as eth1
RM520N-GL just connected to SOC eth1.
just remove these lines

Reply:
> why do this? eth1 should up and nothing related to RM520N-GL RM520N-GL never register as eth1 RM520N-GL just connected to SOC eth1. just remove these lines

Thank you for your feedback! The issue was on my side. Previously, this was to mimic the official system behavior, where `eth1` acquires the IP, and there’s no `eth2`. However, in OpenWrt, after flashing and rebooting, if `eth1` is enabled, the device registers an `eth2`, but only `eth2` can acquire the IP.

I’ve since re-cloned and recompiled OpenWrt, and the issue no longer happened. At this time, I found that the cellular module registers as `eth2` (`eth2: register 'cdc_ether' at usb-11200000.usb-1`), and `eth1` can also acquire an IP normally.